Containerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/devto/ruby:2.7.1
+
+USER root
+
+RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo && \
+    dnf install -y bash git ImageMagick iproute less libcurl libcurl-devel \
```

Comment:
Not sure how I feel about Fedora in this context. Personally I like the distro, and Alpine currently doesn't work for us because of the musl vs libc problem with `mini_racer`. But almost all containers I interact with seem to either inherit from scratch (e.g. multi-stage Go, with statically linked binaries), Alpine or Debian. 

I really don't want to start discussing distributions here, my only point is expectations/familiary for external contributors.

Reply:
As long as we provide support and documentation for the containers we are providing our users the distribution inside the container shouldn't be a point of contention IMO.

Reply:
Fair enough. Let's at least keep some of the documentation close to the code, i.e. 

```docker
# Custom Ruby imaged based on Fedora
FROM quay.io/devto/ruby:2.7.1
```

so less experienced users don't need to figure this out by googling what DNF is.

Reply:
:+1: I can do some better documentation before I merge this for sure.